tech.json

```diff
@@ -637,10 +637,13 @@
         },
         {
           "id": 47,
-          "name": "canQuickCrumbleEscape",
+          "name": "canQuickDrop",
           "techRequires": [],
           "otherRequires": [],
-          "note": "The combination of a crumble quick drop, and landing on a lower surface and jumping back over the crumble block before it re-forms."
+          "note": [
+            "Using a momentum conserving turnaround on a crumble block, an opening doorshell, or any other breaking block, to preserve Samus' momentum.",
```

Comment:
Using a tech name implies its a requirement.  It'd be better to redeclare the action.
While these are separate techs that is.